usecases/auth/authorization/rbac/manager.go

```diff
@@ -287,7 +323,11 @@ func (m *manager) Snapshot() ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
-func (m *manager) Restore(b []byte) error {
+func (m *Manager) Restore(b []byte) error {
+	// restore is only needed when RBAC was enabled during backup
```

Comment:
Note that this function is also used for RAFT snapshots so the comment might be confusing like this

Reply:
What would you suggest?

Reply:
made a proposal in the comment below :)